Figures/IFigure.cs

```diff
@@ -1,24 +1,16 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Figures
+﻿namespace Figures
 {
     public interface IFigure
     {
         int ID {get; set;}
-        double XCoordinate { get; set; }
-        double YCoordinate { get; set; }
-        double Angle { get; set; }
+        double CenterX { get; }
+        double CenterY { get; }
+        Orientation FigureOrientation { get; }
```

Comment:
Why do you need this 'Orientation', if it is not observable to the outside world? I see that you use it inside your classes, but the outside world knows nothing about it - other properties affect how the object is displayed (in our simple case - printed), but this one doesn't. Let's print it as well.